packages/suite-desktop/package.json

```diff
@@ -195,14 +195,16 @@
         "electron-updater": "4.6.5",
         "node-fetch": "^2.6.7",
         "openpgp": "^5.0.0",
+        "react-redux": "7.2.2",
```

Comment:
is it required here? (it will be installed on the main electron process > if you decompile the app it will be present in "node_modules" but not used there)

Reply:
I removed it and squash it. I will add it to devDependencies as a part of https://github.com/trezor/trezor-suite/issues/5537